aries-site/src/examples/components/controls/AccordionExample.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Box } from 'grommet';
+import { Accordion, AccordionPanel } from 'aries-core';
+
+export const AccordionExample = () => {
+  return (
+    <Box width={{ max: 'medium' }}>
+      <Accordion>
+        <AccordionPanel label="Our Company">
+          <Box pad={{ vertical: 'medium' }}>We are HPE.</Box>
+        </AccordionPanel>
+        <AccordionPanel label="Our History">
+          <Box pad={{ vertical: 'medium' }}>
```

Comment:
Little opportunity to create an `<AccordianContent>` sub component since `<Box pad={{ vertical: 'medium' }} >` is being repeated 4 times. Not necessary though.